packages/protocol/contracts/governance/BlockchainParameters.sol

```diff
@@ -75,6 +88,15 @@ contract BlockchainParameters is Ownable, Initializable {
     emit IntrinsicGasForAlternativeFeeCurrencySet(gas);
   }
 
+  /**
+   * @notice Sets the uptime lookback window.
+   * @param window New window.
+   */
+  function setUptimeLookbackWindow(uint256 window) public onlyOwner {
```

Comment:
You should check that the input is within reasonable bounds

Reply:
what would reasonable bounds be for this parameter?

Reply:
The proposed limits, according to the [CIP](https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-0021.md), are [12 (1 minute) - 720 (1 hour)], I'm setting those for now

Reply:
I assume a window of 0 or 1 day should be impossible?

Reply:
Added check as specified on https://github.com/celo-org/celo-proposals/pull/145/files